src/modules/messages.ts

```diff
@@ -1,5 +1,5 @@
-import { DongleConfig } from './DongleDriver'
-import { clamp } from './utils'
+import { DongleConfig } from './DongleDriver.js'
```

Comment:
Do you need the .js imports? I know you need them per the ESM spec, but with moduleResolution bundled we shouldn't need these. A bit weird having .js imports in TS code

Reply:
It's necessary to have them, with these changes, you can now do `import CarplayWeb from 'node-carplay/web'` **without** any bundler and it'll work vanilla

Without the extension, you'd have to hook into the module resolver API and add some hacks